test/integration/event_log_test.rb

```diff
@@ -237,6 +237,34 @@ class EventLogIntegrationTest < ActionDispatch::IntegrationTest
     assert page.has_content?("You do not have permission to perform this action")
   end
 
+  test "record user's login ip address" do
+    visit root_path
+    signin_with(@user)
+
+    ip_address = @user.event_logs.last.trailing_message.split.first
+    assert ip_address.match(/\A[[:alnum:].:]+$/)
+  end
```

Comment:
It would be good if this test could assert on an actual ip address value, but I don't know if it's possible to stub the request ip address somehow so we have a known value to check against.